integration_tests/integration_tests/tests/test_metrics.py

```diff
@@ -0,0 +1,39 @@
+import random
+from datetime import datetime
+
+from data_generator import DATE_FORMAT, generate_dates
+from dbt_project import DbtProject
+
+
+def test_metrics(dbt_project: DbtProject):
+    data1 = [
+        {"updated_at": date.strftime(DATE_FORMAT)}
+        for date in generate_dates(base_date=datetime.now())
+        for _ in range(random.randint(-5, 20))
+    ]
+    data2 = [
+        {"created_at": date.strftime(DATE_FORMAT)}
+        for date in generate_dates(base_date=datetime.now())
+        for _ in range(random.randint(0, 20))
+    ]
+    dbt_project.seed(data1, "first_metrics_table_seed")
```

Comment:
Should we implement `seed_bulk()` for performance?
Alternatively we can just implement the various data seeders per adapter.